### DISCUSSION TITLE
Why `bits-ui` is installed in `dependencies` and not `devDependencies`?

### DISCUSSION THREAD

**Discussion by babakfp** [huntabyte/shadcn-svelte#424] [General]
![image](https://github.com/huntabyte/shadcn-svelte/assets/44144724/2b65a7ce-4fde-4589-9b11-b444014fc031)

Is this a bug?

**Comment by AdrianGonz97** [comment]
No, it's not a bug. It also doesn't matter if it's installed in `dependencies` or `devDependencies` because everything will be bundled anyway during build. If you prefer to have it in `devDependencies`, feel free to move it in your project.

**Reply by babakfp** [reply]
Aha, if there is no purpose in putting it in `dependencies`, then I think it would be nice to put it in `devDependencies` when installing, to keep things consistent.
